xbmc/settings/MediaSettings.cpp

```diff
@@ -220,6 +221,7 @@ bool CMediaSettings::Save(TiXmlNode *settings) const
   XMLUtils::SetBoolean(pNode, "nonlinstretch", m_defaultVideoSettings.m_CustomNonLinStretch);
   XMLUtils::SetInt(pNode, "stereomode", m_defaultVideoSettings.m_StereoMode);
   XMLUtils::SetInt(pNode, "centermixlevel", m_defaultVideoSettings.m_CenterMixLevel);
+  XMLUtils::SetInt(pNode, "tonemapmethod", m_defaultVideoSettings.m_ToneMapMethod);
```

Comment:
what about saving the tonemap param as well?

Reply:
It can be done but I don't think it's convenient. Normally only is need to ajust tonemap param for a specific video. While the tonemap method can be a personal preference for all media library items.